documentation/docs/tutorials/manual_peering.md

```diff
@@ -17,7 +30,7 @@ will keep reconnecting with known peers until it succeeds.
 In other words, the only thing that users have to do to be connected via manual peering is to 
 exchange their IP address with port and public key and set that information to known peers of their nodes and machines will do the rest.
 
-## How to Set Known Peers via Config File
+## How to Set Known Peers Via Config File
```

Comment:
```suggestion
## How to Set Known Peers via Config File
```

Reply:
which style are you using?
Above "With" was capitalized which is e.g. APA
via is generally non-capitalized in most common styles.
https://capitalizemytitle.com/style/APA/